### DISCUSSION TITLE
blog/chatgpt-plugin

### DISCUSSION THREAD

**Discussion by giscus** [JoeyWangTW/joey-wang-blog#18] [General]
# blog/chatgpt-plugin

拿到了ChatGPT Plugin的測試名額，做了一個小小的範例，一次次的被ChatGPT Plugin的威力還有超輕鬆的開發流程驚艷

https://joeywang.blog/blog/chatgpt-plugin?fbclid=IwAR3dQZJr_Qgfd3Q9nUlQmPy97QDJzAwwzVwIvHla0DW97KHhSCyzFHwkwY0

**Comment by naive231** [comment]
youtube tutorial is broken.
